pkg/manifests/directory.go

```diff
@@ -1,31 +1,15 @@
 package manifests
 
-import (
-	"fmt"
+// GetBundleFromDir takes a raw directory containg an Operator Bundle and
+// serializes its component files (CSVs, CRDs, other native kube manifests)
+// and returns it as a Bundle
+func GetBundleFromDir(dir string) (*Bundle, error) {
+	loader := NewBundleLoader(dir)
 
-	internal "github.com/operator-framework/api/pkg/internal"
-	"github.com/operator-framework/api/pkg/validation"
-	"github.com/operator-framework/api/pkg/validation/errors"
-
-	"github.com/operator-framework/operator-registry/pkg/registry"
-)
-
-// GetManifestsDir parses all bundles and a package manifest from dir, which
-// are returned if found along with any errors or warnings encountered while
-// parsing/validating found manifests.
-func GetManifestsDir(dir string) (registry.PackageManifest, []*registry.Bundle, []errors.ManifestResult) {
```

Comment:
This is still something the SDK needs for [`run packagemanifests`](https://github.com/operator-framework/operator-sdk/blob/master/internal/olm/operator/manager.go#L74). Will a function that returns a `PackageManifest` and `[]Bundle` exist somewhere else?